### ISSUE TITLE
Doesn't work properly with folders that has Japanese characters in the folder name

### ISSUE THREAD

**Issue by thewan056** [QL-Win/QuickLook#534]
**Describe the bug**
When you use QL to view a file, example image, in a folder that has Japanese characters in the folder name, the image is not viewed properly. It is zoomed in, and going next image or full screen makes it become blank, not showing anything in the window.

**To Reproduce**
1. Open a folder with Japanese characters in the folder name.
2. Press spacebar to preview any image file in that folder.
3. Press left or right to change image, or full-screen button.
4. Image not viewed properly. Full screen doesn't work. Change image/file doesn't work.

**Expected behavior**
Images are viewed normally like when in folders with English names

**Desktop (please complete the following information):**
 - OS Version: Windows 10 1903
 - QuickLook Version: 3.6.5 msi version

**Comment by xupefei** [comment]
Yeah... this is a known issue. The NConverter.exe we are using does not handle Unicode very well. The workaround is to [set your system locale](https://superuser.com/questions/497857/how-to-change-language-for-non-unicode-programs-in-windows-8) to some East Asian county like China or Japan, but note that this workaround is not recommended unless you are dealing with such files every day.
